devel/mos/Portfile

```diff
@@ -0,0 +1,527 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/mongoose-os/mos 2.18.0
+revision            0
+categories          devel
+license             Apache-2
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+description         Mongoose OS command-line tool
+long_description    {*}${description}. Provides device management \
+                    and firmware building capabilities.
+
+depends_build-append \
+                    port:pkgconfig \
+                    port:python38
+
+# Both libftdi0 and libftdi1 are needed to prevent the following errors:
+# libftdi1: "ld: library not found for -lftdi1"
+# libftdi0: "fatal error: 'ftdi.h' file not found"
+
+depends_lib-append  port:libftdi0 \
+                    port:libftdi1 \
+                    port:libusb
+
+# Makefile tries to run `go mod download` with GO111MODULE=off
+# Removes -mod=vendor
+patchfiles          patch-makefile-download.diff
+
+post-patch {
+    reinplace -W ${worksrcpath}/tools/ "s|python3|python3.8|g" \
+        fw_meta.py
+}
+
+configure.env       PYTHON3=${prefix}/bin/python3.8
+build.cmd           make
```

Comment:
this is the default for most builds, and so it usually isn't needed (unless go ports default to some other build.cmd that I am not aware of...

Reply:
Thanks @kencu. You're right that the build.cmd is different. The default is `go build`, so I changed it manually.

https://github.com/macports/macports-ports/blob/f8d549be3cd0cb9bc9c4cdf7a218f7e99b30dce6/_resources/port1.0/group/golang-1.0.tcl#L148